src/operator/tensor/ordering_op-inl.h

```diff
@@ -558,6 +558,202 @@ void TopKImpl(const RunContext &ctx,
   }
 }
 
+template<typename xpu, typename DType>
+size_t TopK_Workspace_Cal(const TBlob& src,
```

Comment:
Better get rid of the underscores in your function names, follow the convention of all other funcs in the file, you could probably rename to `TopKWorkspaceSize`